### PR TITLE
grafana: add bigquery-grafana plugin

### PR DESCRIPTION
**Pull request by jwygoda**
###### Motivation for this change

https://grafana.com/grafana/plugins/doitintl-bigquery-datasource/

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/servers/monitoring/grafana/plugins/bigquery-grafana/default.nix [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{ grafanaPlugin, lib }:
+
+grafanaPlugin rec {
+  pname = "doitintl-bigquery-datasource";
```
Comment (lukegb) [inline pkgs/servers/monitoring/grafana/plugins/bigquery-grafana/default.nix:4]: I would prefer if the pname and the attribute name match - can you change the attribute name to be "doitintl-bigquery-datasource" (and rename the directory)?

**Review by lukegb — CHANGES_REQUESTED** [review]
The commit message should be:

```
grafanaPlugins.doitintl-bigquery-datasource: init at 2.0.1
```

to match the CONTRIBUTING.md guidelines (and make ofBorg build the correct thing)

@ofborg build grafanaPlugins.bigquery-grafana

File: pkgs/servers/monitoring/grafana/plugins/bigquery-grafana/default.nix (hunk 1 above)
Comment (jwygoda) [reply pkgs/servers/monitoring/grafana/plugins/bigquery-grafana/default.nix]: Thanks for the feedback. Changed commit message, attr name and dir name.

File: pkgs/servers/monitoring/grafana/plugins/doitintl-bigquery-datasource/default.nix [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{ grafanaPlugin, lib }:
+
+grafanaPlugin rec {
+  pname = "doitintl-bigquery-datasource";
+  version = "2.0.1";
+  zipHash = "sha256-tZyvER/rxL+mo2tgxFvwSIAmjFm/AnZ0RgvmD1YAE2U=";
+  meta = with lib; {
+    description = "BigQuery DataSource for Grafana";
+    license = licenses.mit;
+    maintainers = with maintainers; [ jwygoda ];
```
Comment (lukegb) [inline pkgs/servers/monitoring/grafana/plugins/doitintl-bigquery-datasource/default.nix:10]: You'll need to add yourself to maintainers/maintainer-list.nix too :)

**Review by lukegb — APPROVED** [review]
Other than my outstanding comment (maintainer-list.nix), this is good to go!

File: pkgs/servers/monitoring/grafana/plugins/doitintl-bigquery-datasource/default.nix (hunk 2 above)
Comment (jwygoda) [reply pkgs/servers/monitoring/grafana/plugins/doitintl-bigquery-datasource/default.nix:10]: done

**Comment by jwygoda** [comment]
@lukegb I've fixed `maintainer-list.nix`, hope it's all right now

**Comment by lukegb** [comment]
Yup, LG; I don't have merge rights, so you'll need to wait for a committer to swing around.

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Pull request merged**